internal/bucket/bucket_test.go

```diff
@@ -69,14 +69,16 @@ func TestOpen(t *testing.T) {
 	}
 
 	for name, tc := range tests {
-		tc := tc
 		t.Run(name, func(t *testing.T) {
 			t.Parallel()
 
 			b, err := bucket.Open(context.Background(), tc.config)
+			// We should definitely close buckets when we are testing!
+			// This throws a lint error which should be ignored!
+			defer b.Close()
 
 			if tc.errMsg != "" {
-				assert.Assert(t, b == nil)
+				assert.Equal(t, b, nil)
```

Comment:
Thank you. However I think this is not an improvement compared to the original version because `nil` is untyped, i.e. not the same as `(*blob.Bucket)(nil)`. More details here: https://go101.org/article/nil.html. You could see errors like the following:

    assertion failed: <nil> (b *blob.Bucket) != <nil> (nil <nil>)

You'd need to write:

    assert.Equal(t, b, (*blob.Bucket)(nil))
    
At that point I much prefer the original:

    assert.Assert(t, b == nil)